Werewolf for Telegram/Werewolf Control/Commands/DevCommands.cs

```diff
@@ -678,11 +678,11 @@ public static void AddDonation(Update u, string[] args)
                             var msg = "";
                             if (wasLocked)
                             {
-                                msg = "GIF Pack unlocked.  Your current donation level is " + p.DonationLevel;
+                                msg = "GIF Pack unlocked. Your current donation level is " + p.DonationLevel + "\n" + "Now you can use /customgif to make your personal pack.";
```

Comment:
Well, I told you the `"\n" + "Now you can use..."` can be put in one string: `"\nNow you can use..."`

I will still merge it though ;)